src/config.yml

```diff
@@ -85,6 +85,7 @@ allow:
 blocked_commands:
   # Disabled commands
   - 'n:b:/time:Server-side time changing is disabled. Please use /ptime to set your own personal time.'
+  - 'n:b:/etime:Server-Side Time changing is disabled please use /ptime instead!'
```

Comment:
/etime is not necessary. All aliases are blocked by default. Please remove this. :)